core/src/test/java/org/elasticsearch/watcher/ResourceWatcherServiceTests.java

```diff
@@ -36,9 +38,9 @@ public void testSettings() throws Exception {
         // checking the defaults
         Settings settings = Settings.builder().build();
         ResourceWatcherService service = new ResourceWatcherService(settings, threadPool);
-        assertThat(service.highMonitor.interval, is(ResourceWatcherService.Frequency.HIGH.interval));
-        assertThat(service.mediumMonitor.interval, is(ResourceWatcherService.Frequency.MEDIUM.interval));
-        assertThat(service.lowMonitor.interval, is(ResourceWatcherService.Frequency.LOW.interval));
+        assertThat(service.highMonitor.interval, is(TimeValue.timeValueSeconds(SensitivityWatchEventModifier.HIGH.sensitivityValueInSeconds())));
```

Comment:
We shouldn't be using SensitivityWatchEventModifier (forbidden API). You might have forgotten to revert this.